### PR TITLE
[BUGFIX] Fix wrong information in code snippets

### PR DESCRIPTION
**Pull request by peterkraume**
The settings `labelChecked` and `labelUnchecked` are only used with render type `checkboxLabeledToggle` and misleading here.

See core for reference:
https://github.com/TYPO3/typo3/blob/main/typo3/sysext/backend/Classes/Form/FormDataProvider/TcaCheckboxItems.php#L86

The error exists also in older versions of the documentation. If desired, I can also create patches for these versions.

### CODE REVIEW

**Comment by brotkrueml** [comment]
@peterkraume Thanks for your contribution. These snippets come from the styleguide extension, where it is also wrong:
https://github.com/TYPO3/styleguide/blob/main/Configuration/TCA/tx_styleguide_elements_basic.php#L1306-L1336
So, maybe you want to create a PR there also.

Regarding the backport, we have an automatic backport via GH actions.

**Comment by brotkrueml** [comment]
Additionally, the snippets are generated automatically from the styleguide extension.

@linawolf Should we merge this PR now or wait until the PR in the styleguide extension is merged and generate the snippets anew?

**Comment by peterkraume** [comment]
I'll create the styleguide PR tomorrow morning.

**Comment by peterkraume** [comment]
TYPO3/styleguide#427 done

**Comment by linawolf** [comment]
I think we should wait for the styleguide PR to be merged. The next regeneration of code-snippets will update this code snippet anyway

**Comment by peterkraume** [comment]
FYI: https://github.com/TYPO3/styleguide/pull/427 has just been merged

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**
